src/components/network/network-table.js

```diff
@@ -50,10 +71,74 @@ genotet.NetworkTable.prototype.create = function(table, edges) {
       {title: 'Weight', data: 'weight'},
       {title: '', data: 'added'}
     ],
+    select: true,
+    dom: '<"row"<"col-sm-12"B>>' +
+      '<"row"<"col-sm-5"l><"col-sm-7"f>>' +
+      '<"row"<"col-sm-12"tr>>' +
+      '<"row"<"col-sm-5"i><"col-sm-7"p>>',
+    buttons: [
+      {
+        text: 'Add',
+        action: function(e, dt, node, config) {
+          var selectedEdge = dt.rows({selected: true}).data()[0];
+          this.signal('addEdge', {
+            source: selectedEdge.source,
+            target: selectedEdge.target,
+            weight: selectedEdge.originalWeight
+          });
+          edgesForTable.forEach(function(edge) {
+            if (edge.id == selectedEdge.id) {
+              edge.added = true;
+            }
+          });
+          // refresh the row
+          dt.row({selected: true}).invalidate();
+          // change the button status
+          dt.buttons(0).enable(false);
```

Comment:
Please try to avoid index 0, 1, ... to make code cleaner if possible. If you really have to use index, add comments to explain what are you selecting.